.templates/php/default.mk

```diff
@@ -0,0 +1,51 @@
+# Please update /.templates/php/default.mk and sync:
+#
+#     source scripts/functions.sh && rsync_files
+#
+SHELL := /usr/bin/env bash
+PHP_SOURCE_FILES = $(shell find . -name "*.php")
+
+### COMMON stuff for all platforms
+
+### Common targets for all functionalities implemented on php
+
+default: .tested
+.PHONY: default
+
+pre-release: update-version update-dependencies
+.PHONY: pre-release
+
+update-version:
+ifdef NEW_VERSION
+# TODO: something here?
+endif
+.PHONY: update-version
+
+update-dependencies:
```

Comment:
I imagine this should be `composer update`

We currently aren't committing `composer.lock` though so I'm not sure it'd be doing anything

Reply:
Can be fine-tune at the first release of any php module